client/blocks/domain-to-plan-nudge/docs/example.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import DomainToPlanNudge from '../';
+import { getCurrentUser } from 'state/current-user/selectors';
+import QuerySites from 'components/data/query-sites';
+
+function DomainToPlanNudgeExample( { siteId } ) {
+	return (
+		<div className="docs__design-assets-group">
+			<h2>
+				<a href="/devdocs/blocks/domain-to-plan-nudge">Domain To Plan Nudge</a>
+			</h2>
```

Comment:
I would maybe add a warning in form of text saying that if you have a real CC and click on the upgrade btn you'll be charged real monies. :D